src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -146,12 +146,13 @@ public boolean equals(Object other) {
      * Sorts all persons in list in ascending order.
      */
     public void sortPersonsInAddressBook(){
-        Collections.sort(internalList, new Comparator<Person>() {
+        Comparator<Person> personComparatorByName = new Comparator<Person>() {
```

Comment:
Suggest to use a lamda expression to clean up the code.

Reply:
Ok done